api/src/db/models/legacy_mixin/user_mixin.py

```diff
@@ -0,0 +1,38 @@
+from datetime import datetime
+
+from sqlalchemy.orm import Mapped, declarative_mixin, mapped_column
+
+
+@declarative_mixin
+class TuserAccountMapperMixin:
+    user_account_id: Mapped[int] = mapped_column(primary_key=True)
+    source_type: Mapped[str]
```

Comment:
Two issues, one is on me for not putting it in the docs right:
1. This should also be a primary key
2. This should be the last column defined (don't know why I put it second)